Volume/Views/MainView/HomeList.swift

```diff
@@ -197,17 +247,28 @@ struct HomeList: View {
 
     var body: some View {
         RefreshableScrollView(onRefresh: { done in
-            switch state {
-            case .loading, .reloading:
-                return
-            case .results(let results):
-                state = .reloading(results)
+            // TODO: add weekly debrief to this switch when the request stops breaking
+            switch (sectionStates.trendingArticles, sectionStates.followedArticles, sectionStates.otherArticles) {
+            case (.results(let trendingArticles), .results(let followedArticles), .results(let otherArticles)):
+                // Allow refresh when all results are done fetching.
+                sectionStates.trendingArticles = .reloading(trendingArticles)
+//                sectionStates.weeklyDebrief = .reloading(weeklyDebrief)
```

Comment:
delete comment

Reply:
keeping this comment for a future PR. Related to the `TODO` comment on line 250.